inventory/inventory.ini

```diff
@@ -0,0 +1,304 @@
+
+# Internal XMPP domain for authenticated services
+xmpp_auth_domain="auth.meet.jitsi"
+
+# XMPP domain for the MUC
+xmpp_muc_domain="muc.meet.jitsi"
+
+# XMPP domain for the internal MUC used for jibri, jigasi and jvb pools
+xmpp_internal_muc_domain="internal-muc.meet.jitsi"
+
+# XMPP domain for unauthenticated users
+xmpp_guest_domain="guest.meet.jitsi"
+
+# Custom Prosody modules for XMPP_DOMAIN (comma separated)
+xmpp_modules=""
+
+# Custom Prosody modules for MUC component (comma separated)
+xmpp_muc_modules=""
+
+# Custom Prosody modules for internal MUC component (comma separated)
+xmpp_internal_muc_modules=""
+
+# MUC for the JVB pool
+jvb_brewery_muc="jvbbrewery"
+
+# XMPP user for JVB client connections
+jvb_auth_user="jvb"
+
+# STUN servers used to discover the server's public IP
+jvb_stun_servers="meet-jit-si-turnrelay.jitsi.net:443"
+
+# Media port for the Jitsi Videobridge
+jvb_port="10000"
+
+# TCP Fallback for Jitsi Videobridge for when UDP isn't available
+jvb_tcp_harvester_disabled="true"
+jvb_tcp_port="4443"
+jvb_tcp_mapped_port="4443"
+
+# A comma separated list of APIs to enable when the JVB is started [default: none]
+# See https://github.com/jitsi/jitsi-videobridge/blob/master/doc/rest.md for more information
+#jvb_enable_apis="rest,colibri"
+
+# XMPP user for Jicofo client connections.
+# NOTE: this option doesn't currently work due to a bug
+jicofo_auth_user="focus"
+
+# Base URL of Jicofo's reservation REST API
+#jicofo_reservation_rest:base_url=http://reservation.example.com
+
+# XMPP user for Jigasi MUC client connections
+jigasi_xmpp_user="jigasi"
+
+# MUC name for the Jigasi pool
+jigasi_brewery_muc="jigasibrewery"
+
+# Minimum port for media used by Jigasi
+jigasi_port_min="20000"
+
+# Maximum port for media used by Jigasi
+jigasi_port_max="20050"
+
+# TODO: not implemented in the jinja2 template (.env.j2)
+# Enable SDES srtp
+#jigasi_enable_sdes_srtp="1"
+
+# Keepalive method
+#jigasi_sip_peep_alive_method="OPTIONS"
+
+# Health-check extension
+#jigasi_health_check_sip_uri="keepalive"
+
+# Health-check interval
+#jigasi_health_check_interval="300000"
+
+# Enable Jigasi transcription
+#enable_transcriptions="1"
+
+# Jigasi will record audio when transcriber is on [default: false]
+#jigasi_transcriber_record_audio="true"
+
+# Jigasi will send transcribed text to the chat when transcriber is on [default: false]
+#jigasi_transcriber_send_txt="true"
+
+# Jigasi will post an url to the chat with transcription file [default: false]
+#jigasi_transcriber_advertise_url="true"
+
+# Credentials for connect to Cloud Google API from Jigasi
+# Please read https://cloud.google.com/text-to-speech/docs/quickstart-protocol
+# section "Before you begin" paragraph 1 to 5
+# Copy the values from the json to the related env vars
+#gc_project_id=""
+#gc_private_key_id=""
+#gc_prvate_key=""
+#gc_client_email=""
+#gc_client_id=""
+#gc_client_cert_url=""
+
+# Enable recording
+#enable_recording="1"
```

Comment:
Der Default für `enable_recording` ist sicher `off`. Spricht etwas dagegen es explizit zu deaktivieren?

Reply:
Wenn die aktiviert ist, sollte jeder die Möglichkeit haben die Aufnahme zu starten.

Die Funktion habe ich jedoch nicht getestet und bewusst hier durch kommentieren deaktiviert. 
Die dafür notwendigen Konfigurationen für Jibri habe ich auch aus dem Default übernommen.

Mehr zur Konfiguration kann hier gefunden werden:
https://jitsi.github.io/handbook/docs/devops-guide/devops-guide-docker#jitsi-broadcasting-infrastructure-jibri-configuration

Wichtig:
Ansible nutzt die Variablen die in den Inventories mitgegeben werden. Diese werden dann in Jinja2 Templates verarbeitet.

Hier der passende Ausschnitt zum Jinja2-Template `.env.j2` für die `.env` Datei

```
# Enable recording
{% if enable_recording is defined %}
ENABLE_RECORDING={{ enable_recording }}
{% else %}
#ENABLE_RECORDING=1
{% endif %}
```